Realm/Realm/Schema/RealmSchema.cs

```diff
@@ -212,12 +210,18 @@ public RealmSchema Build()
             {
                 if (Count == 0)
                 {
-                    throw new InvalidOperationException(
-                        "No RealmObjects. Has linker stripped them? See https://docs.mongodb.com/realm-legacy/docs/dotnet/latest/#linker-stripped-schema");
+                    var message = InteropConfig.Platform switch
+                    {
+                        InteropConfig.UnityPlatform => "Try weaving assemblies again ('Realm' -> 'Weave Assemblies' from the editor or simply make a code change) and make sure you don't have any Realm-related errors in the logs.",
+                        InteropConfig.DotNetPlatform => "Has linker stripped them? See https://docs.mongodb.com/realm-legacy/docs/dotnet/latest/#linker-stripped-schema",
```

Comment:
You may want to link the new documentation about that: [https://docs.mongodb.com/realm/sdk/dotnet/troubleshooting/](https://docs.mongodb.com/realm/sdk/dotnet/troubleshooting/)